src/interface/src/app/treatments/direct-impacts/direct-impacts.component.ts

```diff
@@ -79,4 +85,7 @@ export class DirectImpactsComponent {
   treatmentPlan$ = this.treatmentsState.treatmentPlan$;
 
   showTreatmentPrescription = false;
+
+  // Metric will have metric.active = true if it should be turned on or false if it should be turned off
```

Comment:
Im not sure i understand this comment.
At this point, the metric we are receiving is the active one, we shouldn't be checking anything else.

Reply:
Updated